test/openssl/test_ssl.rb

```diff
@@ -1630,6 +1631,7 @@ def test_ciphers_method_tls_connection
         server_connect(port, cli_ctx) do |ssl|
           assert_equal('TLSv1.2', ssl.ssl_version)
           assert_equal(csuite[0], ssl.cipher[0])
+          ssl.puts('abc'); assert_equal("abc\n", ssl.gets)
```

Comment:
It may be the case that I only have to do this for the TLSv1.3 method, but it doesn't hurt to do it here too.